editor/src/components/inspector/common/css-utils.ts

```diff
@@ -4047,6 +4048,10 @@ export interface ParsedCSSProperties {
   flexGrow: number
   flexShrink: number
   display: string
+  width: CSSNumber | undefined
+  height: CSSNumber | undefined
+  flexBasis: CSSNumber | undefined
+  flexGap: number
```

Comment:
I think there's no `flex-gap` css property. there's a new-ish `gap` property which wors for flex. https://developer.mozilla.org/en-US/docs/Web/CSS/gap

it was introduced in Chrome 70